tiles/src/main/java/com/protomaps/basemap/layers/Places.java

```diff
@@ -25,6 +29,17 @@ public String name() {
 
   private final AtomicInteger placeNumber = new AtomicInteger(0);
 
+  // To have consistent min_zoom and other attr between Natural Earth (low zooms) and OpenStreetMap (high zooms)
+  // we need to store the NE places with a spatial indexes for later joins with OSM
+  private PointIndex<NaturalEarthPlace> ne_populated_places = PointIndex.create();
```

Comment:
After creation inserts are blocked?

Reply:
Verified this works in a build with no errors... as long as NE is added to Planetiler profile before OSM.